Lib/concurrent/futures/process.py

```diff
@@ -713,15 +719,27 @@ def _adjust_process_count(self):
 
         process_count = len(self._processes)
         if process_count < self._max_workers:
-            p = self._mp_context.Process(
-                target=_process_worker,
-                args=(self._call_queue,
-                      self._result_queue,
-                      self._initializer,
-                      self._initargs,
-                      self._max_tasks_per_child))
-            p.start()
-            self._processes[p.pid] = p
+            assert self._safe_to_dynamically_spawn_children or not self._executor_manager_thread, 'https://github.com/python/cpython/issues/90622'
+            self.__spawn_process()
+
+    def __launch_processes(self):
```

Comment:
The use of double leading underscores here seems gratuitous, can we stick to the existing of a single leading underscore for internal methods?